static/js/publisher/release/reducers/index.js

```diff
@@ -0,0 +1,19 @@
+import { combineReducers } from "redux";
+
+const revisions = function(state = {}, action) {
+  switch (action.type) {
+    case "UPDATE_REVISIONS":
```

Comment:
Can we put actions as "constants" at the top of the file so we can export them to be used elsewhere?
```
const UPDATE_REVISIONS = "UPDATE_REVISIONS";
...
case UPDATE_REVISIONS:
...
export { releases as default, UPDATE_REVISONS }
```

The exports can get quite big, but it makes it easier to keep track (I find)

Reply:
Just realised this is the reducers file, these constants should really be in the actions... So disregard for this PR.

Reply:
Yes, that's somewhere on the way of future changes.

Reply:
> Can we put actions as "constants" at the top of the file so we can export them to be used elsewhere?
> The exports can get quite big, but it makes it easier to keep track (I find)

+1, this does create more boilerplate (which redux in general suffers from a bit), but it is safer. You can even namespace your actions by functional parts of your app, to prevent conflicts if you're paranoid, although probably overkill.

e.g. `const UPDATE_REVISIONS = "release/UPDATE_REVISIONS";`

Reply:
> Just realised this is the reducers file, these constants should really be in the actions... So disregard for this PR.

You can keep constants in their own module (`constants.js`), just to reduce clutter in actions.

Reply:
We already have a constants.js where we keep some strings (like risk names, etc), so may be a good place to move action names as well. This will allow to import them from just one place rather then each action module separately.